data/attributes.yml

```diff
@@ -30209,6 +30209,22 @@ base_attributes:
   values:
   - vehicle_fluid_type__concentrate
   - vehicle_fluid_type__prediluted
+- id: 3418
+  name: Dance style
+  description: Identifies the specific genre or type of dance
+  friendly_id: dance_style
+  handle: dance-style
+  values:
+  - dance_style__ballet
+  - dance_style__tap dance
+  - dance_style__jazz
+  - dance_style__ballroom
+  - dance_style__hip_hop
+  - dance_style__ Salsa
+  - dance_style__ flamenco
+  - dance_style__ contemporary
```

Comment:
A few unnecessary spaces, missing connectors, etc. in these friendly ids. Salsa should be in lower case in the friendly id.

Reply:
fixed